src/global/shared-types/img.types.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * This interface is used to specify a path to an image,
+ * along with related properties, like alt text.
+ * @public
+ */
+export interface Img {
+    /**
+     * The path to the image file.
+     */
+    src: string;
+
+    /**
+     * The alternative text of the image, used to improve accessibility.
+     */
+    alt?: string;
```

Comment:
Should this not be required?

Reply:
It's required on the `img` element. Making it required would be to "highly recommend" it since if a user dont want it they will simply put an empty string in there :smile: